src/api/test/functional/attributes_test.rb

```diff
@@ -131,6 +126,10 @@ def test_create_type
     assert_response :success
     get "/attribute/TEST/Dummy/_meta"
     assert_response :success
+    login_Iggy
+    delete "/attribute/TEST/Dummy/_meta"
+    assert_response 403
+    login_adrian
```

Comment:
Since there is only one way inwhich a 403 response can arise, I dont think we should test the error message. We only care that the 403 response was returned. The content of the error message is irrelevant to a unit test. 

If however there were multiple reasons why a 403 response could be returned then I think we should test the error message to make sure we have a test for each case.

The ideal situation is to use a translations config file so we don't have to include the exact error message in the test i.e.
```ruby
assert_match(I18n.translate('attribute.errors.invalid_request_data'), @response.body)
```
or
```ruby
assert_match(I18n.translate('attribute.errors.incorrect_password'), @response.body)
```

But of course since we don't use translations thats not possible. So I guess the only option is to write the error message in the test.